Content.Server/_Starlight/Construction/Completions/TransformMech.cs

```diff
@@ -88,6 +98,12 @@ public void PerformAction(EntityUid uid, EntityUid? userUid, IEntityManager enti
                 containerSystem.Remove(equipment, equipmentContainer);
                 containerSystem.Insert(equipment, mechComp.EquipmentContainer);
             }
+            while (mechComp.PilotSlot.ContainedEntity == null && pilotContainer.ContainedEntities.Count > 0)
+            {
+                var pilot = pilotContainer.ContainedEntities[0];
+                mechSys.TryEject(uid);
+                mechSys.TryInsert(mech, pilot);
+            }
```

Comment:
to be honest instead of unneccesary `while` you can use just `if`, because only one pilot can be in mech.


```suggestion
            if (mechComp.PilotSlot.ContainedEntity == null && pilotContainer.ContainedEntities.Count > 0)
            {
                mechSys.TryEject(uid);
                mechSys.TryInsert(mech, pilotContainer.ContainedEntities[0]);
            }
```

Reply:
See above

Reply:
> See above

this change is not only about starlight comments

Reply:
True. My thought was "just in case that changes", but really, why would it? Agreed.